tests/admin_filters/models.py

```diff
@@ -31,6 +31,11 @@ class Book(models.Model):
     is_best_seller = models.BooleanField(default=0, null=True)
     is_best_seller2 = models.NullBooleanField(default=0)
     date_registered = models.DateField(null=True)
+    availability_choices = (
+        (False, 'paid'),
+        (True, 'free'),
+    )
+    availability = models.BooleanField(choices=availability_choices, null=True)
```

Comment:
We don't reuse `availability_choices` anywhere so there is no need for a separate variable.

Reply:
Removed the separate variable for the same.